Java/src/main/java/net/hypixel/api/util/Utilities.java

```diff
@@ -15,7 +32,10 @@ public static UUID uuidFromString(String uuidStr) {
         if (uuidStr.contains("-")) {
             return UUID.fromString(uuidStr);
         } else {
-            return UUID.fromString(uuidStr.substring(0, 8) + "-" + uuidStr.substring(8, 12) + "-" + uuidStr.substring(12, 16) + "-" + uuidStr.substring(16, 20) + "-" + uuidStr.substring(20, 32));
+            return UUID.fromString(
+                uuidStr.substring(0, 8) + "-" + uuidStr.substring(8, 12) + "-" + uuidStr
+                    .substring(12, 16) + "-" + uuidStr.substring(16, 20) + "-" + uuidStr
+                    .substring(20, 32));
```

Comment:
```suggestion
            uuidStr = uuidStr.substring(0, 8) + "-"
	              + uuidStr.substring(8, 12) + "-"
	              + uuidStr.substring(12, 16) + "-"
		      + uuidStr.substring(16, 20) + "-"
		      + uuidStr.substring(20, 32))

            return UUID.fromString(uuidStr);
```
Seems you're looking to clean the formatting

Reply:
Must've accidentally ran the formatter on that file, but sure why not